idflakies-core/src/main/java/edu/illinois/cs/dt/tools/detection/detectors/AlphabeticalDetector.java

```diff
@@ -0,0 +1,43 @@
+package edu.illinois.cs.dt.tools.detection.detectors;
+
+import java.io.File;
+import java.util.List;
+
+import edu.illinois.cs.dt.tools.detection.DetectionRound;
+import edu.illinois.cs.dt.tools.detection.DetectorUtil;
+import edu.illinois.cs.dt.tools.detection.TestShuffler;
+import edu.illinois.cs.dt.tools.detection.filters.ConfirmationFilter;
+import edu.illinois.cs.dt.tools.detection.filters.UniqueFilter;
+import edu.illinois.cs.dt.tools.runner.InstrumentingSmartRunner;
+import edu.illinois.cs.testrunner.data.results.TestRunResult;
+import edu.illinois.cs.testrunner.runner.Runner;
+
+public class AlphabeticalDetector extends ExecutingDetector {
+    final List<String> tests;
+    private TestRunResult origResult;
+    private TestRunResult lastAlphabeticalResult;
```

Comment:
I don't think you actually needs these last results for AlphabeticalDetector. I believe these last results were used to keep track of what random order was tried previously, and since this detector only has one valid order, it doesn't really need to track that.